Apple/App/Menu/MenuView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  MenuView.swift
+//  App
+//
+//  Created by Thomas Stubblefield on 5/13/23.
+//
+
+import SwiftUI
+
+struct MenuView: View {
+    @State private var isToggled = false
+    @ObservedObject var tunnel: Tunnel
+
+    private func start() {
+        
+        do {
+            try tunnel.start()
+        } catch {
+            print(error)
+        }
+    }
+
+    private func stop() {
+        tunnel.stop()
+    }
+
+    private func configure() {
+        Task { try await tunnel.configure() }
+    }
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text("Burrow")
+                    .fontWeight(.bold)
+
+                Spacer()
+                Toggle("", isOn: $isToggled)
+                    .toggleStyle(SwitchToggleStyle(tint: .blue))
+                    .onChange(of: isToggled) { value in
+                        if value {
+                            start()
+                        } else {
+                            stop()
+                        }
+                        print("Toggle value: \(value)")
+                    }
+            }
+            Divider()
+            switch tunnel.status {
+            case .permissionRequired:
```

Comment:
It feels like the request permission portion could be moved into its own view, to keep each view small